templates/.github/workflows/auto-merge.yml

```diff
@@ -0,0 +1,21 @@
+name: Auto merge
+
+on:
+  check_suite:
+    types:
+      - completed
+
+jobs:
+  auto-merge:
```

Comment:
Let's make sure we do not run on any forks:

```
if: github.event.pull_request.head.repo.full_name == github.repository
```

https://github.community/t/how-to-detect-a-pull-request-from-a-fork/18363/3